pkg/storage/trb_request_document.go

```diff
@@ -132,11 +138,10 @@ func (s *Store) CreateTRBRequestDocument(ctx context.Context, document *models.T
 	return &retDoc, nil
 }
 
-// DeleteTRBRequestDocument deletes an existing TRBRequestDocument, given its ID
+// DeleteTRBRequestDocument sets the deleted_at property on an existing TRBRequestDocument, given its ID
 func (s *Store) DeleteTRBRequestDocument(ctx context.Context, id uuid.UUID) (*models.TRBRequestDocument, error) {
 	const trbRequestDocumentDeleteSQL = `
-		DELETE
-		FROM trb_request_documents
+		UPDATE trb_request_documents SET deleted_at = NOW()
```

Comment:
Same comment as above on using `CURRENT_TIMESTAMP` instead of `NOW()`.

Reply:
37a62e01a